### PR TITLE
refs task #12449 Social login

### PR DESCRIPTION
**Pull request by hungtb95**
https://edu-redmine.sun-asterisk.vn/issues/12449

### CODE REVIEW

**Comment by hungtb95** [comment]
ready

File: app/SocialAccount.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class SocialAccount extends Model
+{
+    protected $fillable = [
+    	'user_id',
+    	'provider_user_id',
+    	'provider'
```
Comment (BunnyPi04) [inline app/SocialAccount.php]: convention
https://viblo.asia/p/cac-convention-thuong-gap-khi-moi-bat-dau-training-php-tai-framgia-bJzKmgq6l9N#_27-mang-va-chuoi-12

File: app/Http/Controllers/Auth/SocialAuthController.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Controllers\Auth;
+
+use App\Http\Controllers\Controller;
+use App\Services\SocialAccountService;
+use Illuminate\Http\Request;
+use Laravel\Socialite\Facades\Socialite;
```
Comment (tranducquoc) [inline app/Http/Controllers/Auth/SocialAuthController.php]: package này dùng làm gì em

Comment (hungtb95) [reply app/Http/Controllers/Auth/SocialAuthController.php]: dạ package này hỗ trợ authentication Facebook, Twitter, LinkedIn, Google, GitHub, GitLab anh ạ

**Pull request merged**